src/dune_rules/inline_tests.ml

```diff
@@ -281,7 +283,10 @@ include Sub_system.Register_end_point (struct
       Compilation_context.create () ~super_context:sctx ~expander ~scope
         ~obj_dir ~modules ~opaque:(Explicit false) ~requires_compile:runner_libs
         ~requires_link:(lazy runner_libs)
-        ~flags:(Ocaml_flags.of_list [ "-w"; "-24"; "-g" ])
+        ~flags:
```

Comment:
Now that this is long enough that it goes across 3 lines, it would be more readable (imho) to use a temporary variable before the call to `Compilation_context.create` like

```
let flags = Ocaml_flags.append_common. ... in
Compilation_context.create () ~super_context:sctx ... ~flags ... <etc.>
```

Of course, this may just be a matter of taste. Feel free to disregard if you disagree on whether it's more readable.